test/OpenIddict.Server.Tests/Internal/OpenIddictServerProviderTests.cs

```diff
@@ -1525,7 +1525,7 @@ public async Task ProcessSignoutResponse_CustomPublicParametersAreAddedToLogoutR
             Assert.Equal("value", (string) response["custom_string_parameter"]);
         }
 
-        private static TestServer CreateAuthorizationServer(Action<OpenIddictServerBuilder> configuration = null)
+        private static TestServer CreateAuthorizationServer(Action<OpenIddictServerBuilder> configuration = null, bool ignoreEndpointPermission = true)
```

Comment:
I don't really like this field injection, but I didn't find better solution for now. Any suggestion @PinpointTownes ? Thanks

Reply:
@ahanoff why not use `OpenIddictServerOptions.IgnoreEndpointPermissions()`

Reply:
@ahanoff a simple (and quite elegant) option consists in directly configuring the underlying `OpenIddictServerOptions` using the `Configure` method (which is what `OpenIddictServerBuilder` does for you):

```csharp
builder.Configure(options => options.IgnoreEndpointPermissions = false);
```